### ISSUE TITLE
Application Cache empty in Ice Cream Sandwich and Jellybean

### ISSUE THREAD

**Issue by srisom** [forcedotcom/SalesforceMobileSDK-Android#141]
Our Android mobile app (Salesforce SDK 1.2) has been running live in the field both on and offline (as required) on Android version 2.2 with no issues.

We are due an upgrade and test devices for ICS and Jellybean are working online but we can no longer start it up offline.

I have done some initial investigation running the app up on emulators for Android versions 2.2, 2.3.3 and 4.1.  I note that the Application cache size on startup is 781312 on both 2.2 and 2.3.3.  Viewing this through a sql lite browser shows that the resources are all present.  However on 4.1, the application cache is only 22525 and appears to be only metadata (ie browsing it shows the structure but it is empty).

To sum up, if we run the app up offline on versions 4.0 and 4.1 we get an error 'the url could not be found' whereas the app runs successfully on prior versions.

Are there any differences in the implementation that we need to know in order to keep our offline capabilities ?
